src/SFA.DAS.Reservations.Web/AppStart/AuthorizationService.cs

```diff
@@ -56,28 +72,28 @@ private static void ProviderOrEmployerAssertion(AuthorizationPolicyBuilder polic
             {
                 var hasUkprn = context.User.HasClaim(claim =>
                     claim.Type.Equals(ProviderClaims.ProviderUkprn));
-                var hasDaa = HasDaaClaim(context);
-
-                
+                var hasDaa = HasValidServiceClaim(context);
 
                 var hasEmployerAccountId = context.User.HasClaim(claim =>
                     claim.Type.Equals(EmployerClaims.AccountsClaimsTypeIdentifier));
                 return hasUkprn && hasDaa || hasEmployerAccountId;
             });
         }
 
-        private static bool HasDaaClaim(AuthorizationHandlerContext context)
+        private static bool HasValidServiceClaim(AuthorizationHandlerContext context)
         {
-            var hasDaaClaim = context.User.HasClaim(claim =>
+            var validClaimsList =  Enum.GetNames(typeof(ServiceClaim)).ToList();
+            var hasValidClaim = context.User.HasClaim(claim =>
                 claim.Type.Equals(ProviderClaims.Service) &&
-                claim.Value.Equals(ProviderDaa));
+                validClaimsList.Any(x => claim.Value.Equals(x)));
 
-            if (!hasDaaClaim)
+            if (!hasValidClaim)
             {
-                hasDaaClaim = context.User.FindAll(ProviderClaims.Service)
+                // It is looking for any service claim - without verifying the value.
+                hasValidClaim = context.User.FindAll(ProviderClaims.Service)
                     .Select(c => c.Value).ToList().Any();
             }
-            return hasDaaClaim;
+            return hasValidClaim;
```

Comment:
no sure I understand this, we look for a valid claim and if we don't find it we accept any service claim as valid?